sslyze/scanner.py

```diff
@@ -264,7 +277,8 @@ def get_results(self) -> Iterable[ServerScanResult]:
                         result = implementation_cls.result_for_completed_scan_jobs(
                             server_info, list(completed_scan_jobs)
                         )
-                        server_scan_results[scan_cmd] = result
+                        scan_cmd_str = cast(str, scan_cmd)
+                        server_scan_results = dataclasses_replace(server_scan_results, **{scan_cmd_str: result})
```

Comment:
In this case let's just not make ScanCommandResults frozen - I think that's fine.

Reply:
Did this right now.